stores/evga.py

```diff
@@ -90,43 +92,113 @@ def login(self, username, password):
             selenium_utils.wait_for_page(
                 self.driver, "EVGA - Intelligent Innovation - Official Website", 300
             )
-            pickle.dump(
-                self.driver.get_cookies(), open("evga-cookies.pkl", "wb")
-            )  # save cookies
+            pickle.dump(self.driver.get_cookies(), open ("evga-cookies.pkl", "wb"))  # save cookies
 
         log.info("Logged in!")
 
-    def buy(self, delay=5, test=False):
+    def buy(self, delay=5, test=False, model=""):
+        selector = '//a[@id="LFrame_btnAddToCart"]'
         if test:
+            model_name = "test"
+            url = "https://www.evga.com/products/product.aspx?pn=08G-P4-3289-KR"
             self.driver.get(
-                "https://www.evga.com/products/ProductList.aspx?type=0&family=GeForce+16+Series+Family&chipset=GTX+1660"
+                url
             )
             selenium_utils.wait_for_page(
                 self.driver,
-                "EVGA - Products - Graphics - GeForce 16 Series Family - GTX 1660",
+                "EVGA - Products - EVGA GeForce RTX 2080 SUPER FTW3 HYDRO COPPER GAMING, 08G-P4-3289-KR, 8GB GDDR6, RGB LED, iCX2 Technology, Metal Backplate - 08G-P4-3289-KR",
             )
         else:
-            self.driver.get(
-                "https://www.evga.com/products/productlist.aspx?type=0&family=GeForce+30+Series+Family&chipset=RTX+3080"
-            )
-            selenium_utils.wait_for_page(
-                self.driver,
-                "EVGA - Products - Graphics - GeForce 30 Series Family - RTX 3080",
-            )
+            if model == 'ftw3':
```

Comment:
Could we store these as a dict instead of having these if/else blocks?

Reply:
Just picking up python today.  Will try.